spec/takeaway_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'takeaway'
+describe CustOrder do
+  
+  describe '#menulist functions' do
+    it { is_expected.to respond_to(:dishlection) }
+    it { is_expected.to respond_to(:show_menu) }
+    it 'Should show menu' do
+      expect(subject.show_menu).to eq([])
+    end
+    it 'Should select correct dish' do
+      expect(subject.dishlection("Chicken Gyoza 5pcs")).to eq([{ :platter => "Chicken Gyoza 5pcs", :price => 4.5, :quantity => 0 }])
```

Comment:
I'm a bit confused as to why you want this method to return a hash that includes `quantity => 0` which then never seems to get updated.